includes/emails/class-wc-email-customer-processing-order.php

```diff
@@ -56,7 +56,7 @@ public function __construct() {
 		 * @return string
 		 */
 		public function get_default_subject() {
-			return __( 'Your {site_title} order receipt from {order_date}', 'woocommerce' );
+			return __( 'Your {site_title} order is now complete', 'woocommerce' );
```

Comment:
Noticed this in my testing. The order is 'processing' status not 'complete' status at this point.